packages/forma-36-website/src/components/Navigation.js

```diff
@@ -164,15 +176,20 @@ MenuListItem.defaultProps = {
   hierarchyLevel: 0,
 };
 
-const MenuList = ({ menuItems, currentPath, hierarchyLevel }) => {
+const MenuList = ({
+  menuItems,
+  currentPath,
+  hierarchyLevel,
+  isHidden = false,
+}) => {
   const activeRef = useRef(null);
   useEffect(() => {
     if (activeRef.current) {
       activeRef.current.scrollIntoView({ block: 'center' });
     }
   }, []);
   return (
-    <ul className={styles.list}>
+    <ul css={[styles.list, isHidden && styles.hidden]}>
```

Comment:
Same question here that @Lelith did in the other PR

Can't we use emotion’s `cx` here?

Reply:
Ohh didn't see we changed to className on the website on the v4, I copied the code from the master, and we were using the css prop there. Will update on this one, but not sure if needed on that other PR, as I replied there.